host_source.go

```diff
@@ -130,6 +130,8 @@ type HostInfo struct {
 	state            nodeState
 	schemaVersion    string
 	tokens           []string
+
+	untranslatedConnectAddress net.IP
```

Comment:
It would be the longest name in the variable group and would cause gofmt to add more spaces for other variables. I wanted to avoid that so that merging with upstream will be easier if there are any changes in the main variable group.

If you have an idea for a name that is short enough to fit in the previous block, but is also equally descriptive, then I'm all ears :)

Reply:
I think that adding more spaces is something git can easily resolve in case of a conflict.

Reply:
OK. In that case, I moved it under connectAddress.